index.php

```diff
@@ -350,6 +350,7 @@ function greska_u_modulima() {
 <head>
 	<title><?=$naslov?></title>
 	<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+	<link rel="stylesheet" href="static/css/priznavanje.css" type="text/css">
```

Comment:
Pošto je priznavanje.css specifičan za modul studentska/priznavanje, ja bih css includao iz tog modula a ne iz index.php jer bi se vremenom index.php napunio tagovima za sve module

Reply:
Kada je riječ o CSS-u, mislim da bi trebalo generisati neki SCSS fajl, koji bi bio iskompajliran u CSS, i to sve da se strpa u jedan, radi brzine očitavanja tj. manjeg broja requesta prema serveru.


Napraviti konvenciju da se "main wrapper" zove u odnosu na zadani modul, te unutar njega možete deklarisati klase po želji.